packages/types/src/credentials.ts

```diff
@@ -26,4 +28,8 @@ export interface Credentials {
   readonly expiration?: Date;
 }
 
+/**
+ * @deprecated use auth.ts::Credentials
```

Comment:
```suggestion
 * @deprecated use auth.ts::AwsCredentialIdentityProvider
```

Reply:
I'm going to remove the duplication in auth.ts and only export non-conflicting types from both